community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,197 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: microcks-operator.v0.1.0
+  namespace: REPLACE_NAMESPACE
```

Comment:
nit: please use `namespace: placeholder` to follow conventions set by other operators. It's purely cosmetic since OLM replaces this.

Reply:
Change `REPLACE_NAMESPACE` by `placeholder` in commit e49983af1e6a18db2c377218a87eab53b9235bd7